@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -418,10 +429,10 @@
   .navds-combobox__list-item--selected {
     background-color: selecteditem;
     color: selecteditemtext;
-  }
 
-  .navds-combobox__list-item--selected > * {
```

Comment:
<img width="291" alt="Screenshot 2024-11-07 at 11 27 38" src="https://github.com/user-attachments/assets/3c1b2646-f5ba-44ca-a10b-6e7f236251e4">

Text is not readable now in force-color mode